docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -142,289 +164,288 @@ <h5>
-          <td>Delete any library versions that should not be loaded or use LD_LIBRARY_PATH to load libraries from <code>/opt/google/lib</code>.</td>
+          <td>460</td>
+          <td>Possibility of seg fault in QDateWrapper</td>
+          <td>No current work around.</td>
         </tr>
         <tr>
-          <td>340</td>
-          <td>GE Fusion Terrain is black</td>
+          <td>456</td>
+          <td>Inconsistent behavior of vector layers after upgrade</td>
           <td>No current work around.</td>
         </tr>
         <tr>
-          <td>342</td>
-          <td>Fusion crashes when opening an unsupported file type</td>
-          <td>Re-open Fusion and avoid opening unsupported file types.</td>
+          <td>453</td>
+          <td>Improve `check_server_processes_running` detection for uninstall</td>
+          <td>No current work around.</td>
         </tr>
         <tr>
-          <td>343</td>
-          <td>gefusion: File ->open->*.kiasset*,*.ktasset*,*.kip does not work</td>
-          <td>kip is not a supported format. Void opening files with .kip extension.</td>
+          <td>448</td>
+          <td>Out of Memory issues</td>
+          <td>Use a system that has more than 4GB RAM.</td>
         </tr>
         <tr>
-          <td>380</td>
-          <td>Provider field in resource-view is blank</td>
-          <td>Open the individual resource to see the provider.</td>
+          <td>445</td>
+          <td>Path to tutorial source volume in gee_test instructions is different from path used in installers</td>
+          <td>Use <code>/opt/google/share/tutorials</code>.</td>
         </tr>
         <tr>
-          <td>401</td>
-          <td>GEE commands are not in the path for sudo. </td>
-          <td>Specify the full path when running commands or add <code>/opt/google/bin</code> to the path for all users, including the super user.</td>
+          <td>444</td>
+          <td>Fusion installer does not upgrade the asset root on RHEL 7</td>
+          <td>Upgrade the asset root manually by running the command that is printed when you try to start the Fusion service.</td>
         </tr>
         <tr>
-          <td>402</td>
-          <td>Provider manager window locked to main window.</td>
+          <td>442</td>
+          <td>Multiple database pushes after upgrade don't report a warning</td>
           <td>No current work around.</td>
         </tr>
         <tr>
-          <td>403</td>
-          <td>Missing Close button on system manager window in RHEL 7</td>
-          <td>Right-click the title bar and select Close.</td>
+          <td>439</td>
+          <td>Uninstalling Fusion without stopping it results in unexpected error message</td>
+          <td>Ignore that error message.</td>
         </tr>
         <tr>
-          <td>404</td>
-          <td>Opaque polygons in preview.</td>
+          <td>437</td>
+          <td>Rebooting VM while it is building resources results in a corrupted XML</td>
```

Comment:
Does this occur with VMs or should we say just "Rebooting Fusion server while..."?

Reply:
I am unsure. Seems like it may happen with everything. The issue itself is not specific.

Reply:
Changed it to your suggestion, as the issue doesn't specifically mention VMs.